openshift/template.yaml

```diff
@@ -394,39 +401,42 @@ objects:
   - apiVersion: batch/v1
     kind: Job
     metadata:
-      name: superuser-job
+      name: superuser-job-1
     spec:
       parallelism: 1
       completions: 1
       activeDeadlineSeconds: 1800
       backoffLimit: 3
       template:
         metadata:
-          name: superb-job
+          name: superuser-job-1
         spec:
           containers:
-            - name: superb-job
+            - name: superuser-job-co
```

Comment:
Can we use `superuser-job-1` to be consistent?

Reply:
Ack.